CarpetLib/src/prolongate_3d_dgfe_rf2.cc

```diff
@@ -171,7 +171,7 @@ void prolongate_3d_dgfe_rf2(
 
     // Loop over fine region
     ptrdiff_t const j = 0;
-#pragma omp parallel for collapse(2)
+    // #pragma omp parallel for collapse(2)
```

Comment:
this should not be committed in this form. Rather the whole pragma should be removed.